config/initializers/govuk_notify.rb

```diff
@@ -30,9 +30,11 @@
   session_clinic_initial_invitation_ryg: "fc99ac81-9eeb-4df8-9aa0-04f0eb48e37f",
   session_clinic_subsequent_invitation_ryg:
     "eee59c1b-3af4-4ccd-8653-940887066390",
+  session_clinic_initial_invitation_rt5: "e1b6a2f6-728a-4de3-88ec-40194b354eac",
```

Comment:
Do we also need a `session_clinic_subsequent_invitation_rt5` template?

(Also would it possible to keep this list in alphabetical order).